bookmarks/services/importer.py

```diff
@@ -20,11 +21,11 @@ class ImportResult:
     failed: int = 0
 
 
-def import_netscape_html(html: str, user: User):
+def import_netscape_html(html: str, user: User, tags_from_folders: str):
```

Comment:
The form data for checkboxes is `"on"` (true) / `""` (false). Does Django convert it into `True`/`False` if it's declared `boolean`? I'm not that proficient in Django.

Reply:
No, my concern is more that this part of the application should be free from frontend semantics. It would be enough to convert into a boolean when calling this function: `request.POST['tags_from_folders'] == 'on'`